environments/openshift/jaeger.jsonnet

```diff
@@ -0,0 +1,93 @@
+local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
+
+local app =
+  (import '../kubernetes/jaeger.libsonnet') + {
+    jaeger+:: {
+      namespace:: '${NAMESPACE}',
+      image:: '${IMAGE}:${IMAGE_TAG}',
+      replicas:: '${{REPLICAS}}',
+
+      queryService+: {
+        metadata+: {
+          annotations+: {
+            'service.alpha.openshift.io/serving-cert-secret-name': 'querier-tls',
```

Comment:
Not used to the naming conventions used in Observatorium, so, just wanted to double check that this is a new secret, and not being reused by some other service.

Reply:
This is actually an OpenShift thing, not an Observatorium one. This will make the OpenShift serving certs operator write a serving TLS certificate into this secret with the service IP and DNS name as valid SAN.

Reply:
Right, I'm more interested in the name itself (`querier-tls`). My first choice would have been `jaeger-query-tls`, but I'm sure there's a reason to be named like that :)

Reply:
Ah sorry my bad. I think it would benefit from a more descriptive name as well.

Reply:
querier-tls is still a copy-paste mistake :upside_down_face: 
I'll fix that!